test/units/parsing/vault.yml

```diff
@@ -0,0 +1,6 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
This should probably be in `test/units/parsing/fixtures/` to match the other parsing unit tests.

Reply:
Done.